packages/simplebar/src/simplebar.js

```diff
@@ -381,19 +381,17 @@ export default class SimpleBar {
       this.el.addEventListener('mouseenter', this.onMouseEnter);
     }
 
-    [
-      'mousedown',
-      'click',
-      'dblclick',
-      'touchstart',
-      'touchend',
-      'touchmove'
-    ].forEach(e => {
+    ['mousedown', 'click', 'dblclick'].forEach(e => {
+      this.el.addEventListener(e, this.onPointerEvent, true);
+    });
+
+    ['touchstart', 'touchend', 'touchmove'].forEach(e => {
```

Comment:
If `preventDefault` is removed there is no need for non-pasive listeners
https://github.com/Grsmto/simplebar/blob/f95c9f8eca614797433cb3b03b35a907aeb39265/packages/simplebar/src/simplebar.js#L722-L724

Reply:
This is not correct. React-Select from demo is not working without `preventDefault`

Reply:
So you think no change is needed?

Reply:
Yeah. Think only css fix was needed. There is no warning in Chrome and Firefox now after events were separated.